ioctl/client.go

```diff
@@ -397,6 +426,10 @@ func (c *client) WriteHdWalletConfigFile(mnemonic string, password string) error
 	return nil
 }
 
+func (c *client) PackABI(name string, args ...interface{}) ([]byte, error) {
+	return c.xrc20ABI.Pack(name, args...)
```

Comment:
xrc20abi is more like a const, instead of a parameter of client

Reply:
`xrc20ABI.Pack(name, args...)`  is used in other xrc20 commands: xrc20_allowance、 xrc20_approve ....
so make this function. 
It seems that  xrc20abi has nothing to do with client. If so, I will put it into package of xrc20.